src/coreclr/vm/frozenobjectheap.cpp

```diff
@@ -0,0 +1,166 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "frozenobjectheap.h"
+#include "memorypool.h"
```

Comment:
```suggestion
#include "memorypool.h"
```
Is it actually needed in the current implementation?

Reply:
Nope, it was a left-over, now it relies on VirtualAlloc to be always zeroe'd